src/main/java/stirling/software/SPDF/model/ApplicationProperties.java

```diff
@@ -202,6 +213,70 @@ public String toString() {
                         + "]";
             }
         }
+
+        public static class OAUTH2 {
```

Comment:
Please update settings.yml.template with this entry to allow population via settings file